dags/datasets/earlyon-centres-registry.py

```diff
@@ -0,0 +1,442 @@
+tmp_folder = Path(Variable.get("tmp_dir")) / dag_id
+filepath = tmp_folder / "data.parquet"
+fields_filepath = tmp_folder / "fields.json"
+schema_change = True
+maker_attr_def = {
+        "service_system_manager": "City of Toronto",
+        "agency": "The agency associated with this location",
+        "loc_id": "Location ID as in CSIS2",
+        "program_name": "The name of the Resource centre ",
+        "languages": "The languages in which service is available at this Location",
+        "french_language_program": "If a French language program is offered",
+        "indigenous_program": "If Indigenous programming offered at this location",
+        "programTypes": "If the program is Drop-In, or Registered, or Virtual",
+        "serviceName": "Additional information about this program",
+        "buildingName": "The Building Name of the location if one exists",
+        "address": "The street address of the location",
+        "full_address": "The full address of the location",
+        "major_intersection": "The closest major intersection to the location address",
+        "ward": "In which ward  (number) is this location",
+        "ward_name": "The ward name of the location",
+        "located_in_school": "Is this Location in a school (Yes/No)?",
+        "school_name": "Name of School EarlyON is located in",
+        "lat": "Latitudinal coordinates of the location address",
+        "lng": "Longitudinal coordinates of the location address",
+        "website": "the location website where parents and caregivers can find more information about the programs",
+        "website_name": "Agency's Website",
+        "website_description": "Agency's Website",
+        "consultant_fullname": "The first and last name of the EarlyOn Consultant assigned to this location",
+        "consultant_phone": "The contact phone number of the EarlyOn Consultant assigned to this location",
+        "email": "The contact email for the EarlyOn Consultant assigned to this location",
+        "contact_fullname": "The contact email for the EarlyOn Consultant assigned to this location",
+        "contact_title": "The contact email of the Location Administrator",
+        "phone": "The contact phone number of the Location Administrator ",
+        "contact_email": "The contact email of the Location Administrator",
+        "dropinHours": "Day of the week, Start and end time, whether program is full time, and frequency for in-person programs",
+        "registeredHours": "Day of the week, Start and end time, whether program is full time, and frequency for in-person registered programs",
+        "virtualHours": "Day of the week, Start and end time, whether program is full time, and frequency for virtual programs",
+        "geometry":"markers geometry",
+        "centre_type": "Centre",
+        "program": "<a id='13650' href='#modal13650_' data-toggle='modal' data-target='#myModal' onclick='fillData(13650)'>101 Spruce St EarlyON Child and Family Centre</a>",
```

Comment:
these are attribute definitions, right?
Do you need "centre_type" and "program" if youre not going to include them in the output dataset?

No big deal - this isnt a show stopper - I just wanted to check

Reply:
I agree, the reason I leave it there so I/new dev can copy sample json object, change data into definition. We also know this is in the source data.

Reply:
ok 👍